ansible_ai_connect/main/settings/base.py

```diff
@@ -541,6 +541,7 @@ def is_ssl_enabled(value: str) -> bool:
 # ------------------------------------------
 CHATBOT_DEFAULT_PROVIDER = os.getenv("CHATBOT_DEFAULT_PROVIDER")
 CHATBOT_DEBUG_UI = os.getenv("CHATBOT_DEBUG_UI", "False").lower() == "true"
+CHATBOT_STREAM = os.getenv("CHATBOT_STREAM", "False").lower() == "true"
```

Comment:
IMO you should really be adding this new environment variable to the pipeline configuration [here](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/model_pipelines/pipelines.py#L187-L210).

Use of environment variables is deprecated; but you could still [make use of it](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/main/settings/legacy.py).. if you really want to.

Reply:
@manstis Thanks. I'll update the code accordingly.